vcs/git/clone_update.go

```diff
@@ -45,7 +45,10 @@ func init() {
 }
 
 func Clone(url, dir string, opt vcs.CloneOpt) (vcs.Repository, error) {
-	clopt := git2go.CloneOptions{Bare: opt.Bare}
+	clopt := git2go.CloneOptions{
+		FetchOptions: &git2go.FetchOptions{},
```

Comment:
I usually try to keep initialization in a central place, but in this case I agree with you that is much more clear. Done.